static/app/components/profiling/ProfilingOnboarding/profilingOnboardingModal.tsx

```diff
@@ -238,6 +246,44 @@ function SelectProjectStep({
   );
 }
 
+function usePublicDSN({
```

Comment:
@eliashussary I would move this into static/app/utils as others will likely benefit from this hook.

Reply:
sure, we can. if we do i'd change it to `useProjectKeys` instead